schemata/cta.yaml

```diff
@@ -0,0 +1,34 @@
+---
+id: cta.json
+"$schema": "http://json-schema.org/draft-04/schema"
+title: Call to action
+description: "An HTML representing a call to action with the verb which defines the action, a tagged price and some measurement data"
+type:
+- object
+properties: 
+  id:
+    "$ref": "base.json#/definitions/uuid"
```

Comment:
I believe there is a way in JSON schema to avoid having to copy all these "base" parameters for blocks:
http://spacetelescope.github.io/understanding-json-schema/structuring.html#extending
This cleanup can be done in a second round though. We have enough testcases to be able to do what without breaking things I think :)